microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -18,20 +18,20 @@ Review this article often.  The Microsoft Edge team updates this article as thin
 | Autoupgrade mixed content images | Edge v88  |  | Non-secure \(HTTP\) references to images are automatically upgraded to HTTPS; if the image is not available over HTTPS, the image download fails. A [Group Policy][DeployedgeMicrosoftEdgePoliciesInsecurecontentallowedforurls] is available to control this feature. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, navigate to the [Chrome Platform Status entry][ChromestatusFeature4926989725073408].  | 
-| HTTP authentication disallowed when third-party cookies are blocked  | Edge v87  |  | Starting with Edge v87, when cookies are blocked for third-party requests, using either the [BlockThirdPartyCookies][DeployedgeMicrosoftEdgePoliciesBlockthirdpartycookies] policy or the toggle in `edge://settings`, HTTP authentication is also disallowed. This change may impact Enterprise Mode [Site List downloads for Internet Explorer mode][DeployedgeEdgeIeModePoliciesConfigureUsingUseEnterpriseModeIeWebsiteListPolicy] if the endpoint hosting the list requires the use of HTTP authentication.  To allow the use of both cookies and HTTP authentication for Enterprise Mode Site List downloads, add a matching URL pattern to the [CookiesAllowedForURLs][DeployedgeMicrosoftEdgePoliciesCookiesallowedforurls] policy.  |
 | Removal of 3DES in TLS  | Edge v93  |  | Starting with Edge v93, support for the TLS_RSA_WITH_3DES_EDE_CBC_SHA cipher suite will be removed. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, navigate to the [Chrome Platform Status entry][ChromestatusFeature6678134168485888]. Additionally, in Edge v93, a compatibility policy will be available to support scenarios that need to retain compatibility with outdated servers. This compatibility policy will become obsolete and stop working in Edge v95. Ensure that you update impacted servers before then. |
 | Restrict private network requests to secure contexts  | Edge v93  |  | Starting with Edge v93, access to resources on local (intranet) networks from pages on the internet requires that those pages be delivered over HTTPS. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, navigate to the [Chrome Platform Status entry][ChromestatusFeature5436853517811712]. Two compatibility policies are available to support scenarios that need to retain compatibility with non-secure pages: [InsecurePrivateNetworkRequestAllowed][DeployEdgeMicrosoftEdgePoliciesInsecurePrivateNetworkRequestAllowed] and [InsecurePrivateNetworkRequestAllowedForUrls][DeployEdgeMicrosoftEdgePoliciesInsecurePrivateNetworkRequestAllowedForUrls]. |
```

Comment:
```suggestion
| Restrict private network requests to secure contexts  | Edge v93  |  | Starting with Edge v93, access to resources on local (intranet) networks from pages on the internet requires that those pages be delivered over HTTPS. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, navigate to the [Chrome Platform Status entry][ChromestatusFeature5436853517811712]. Two compatibility policies are available to support scenarios that need to preserve compatibility with non-secure pages: [InsecurePrivateNetworkRequestAllowed][DeployEdgeMicrosoftEdgePoliciesInsecurePrivateNetworkRequestAllowed] and [InsecurePrivateNetworkRequestAllowedForUrls][DeployEdgeMicrosoftEdgePoliciesInsecurePrivateNetworkRequestAllowedForUrls]. |
```
Another instance where Acrolinx suggested changing "retain" to "keep", but I suggest "preserve".